clifford/taylor_expansions.py

```diff
@@ -0,0 +1,117 @@
+
```

Comment:
Any chance you could add a docstring here, and add `taylor_expansions` to an rst file so that docs generate?